crates/bls12381/src/curves/g1.rs

```diff
@@ -114,23 +138,83 @@ impl<F: PrimeField + PrimeFieldBits> G1Point<F> {
     where
         CS: ConstraintSystem<F>,
     {
-        let cs = &mut cs.namespace(|| "G1::assert_collinear(a, b, c)");
-
-        // compute leftside = (y_a + y_c) * (x_b - x_a)
-        let aycy = a.y.add(&mut cs.namespace(|| "aycy <- a.y + c.y"), &c.y)?;
+        // compute leftside = (y_a - y_c) * (x_b - x_a)
```

Comment:
Minor typo in comment versus implementation

`// compute leftside = (y_a - y_c) * (x_b - x_a)`

implements in ln142:

`aymcy <- a.y + c.y`